model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -105,9 +105,13 @@ class LicenseInfoResolver(
                         location = UNDEFINED_TEXT_LOCATION,
                         appliedCuration = null,
                         matchingPathExcludes = emptyList(),
-                        copyrights = licenseInfo.declaredLicenseInfo.authors.mapTo(mutableSetOf()) {
+                        copyrights = licenseInfo.declaredLicenseInfo.authors.mapTo(mutableSetOf()) { author ->
+                            val statement = "Copyright (C) $author".takeUnless {
+                                author.contains("Copyright", ignoreCase = true)
```

Comment:
Did you intend to not use COPYRIGHT_PREFIX anymore? 
It does becomes unused, doesn't it?

Reply:
Yes, I inlined its only use intentionally.

Reply:
I hadn't noticed the line was removed - so all fine as-is.